### PR TITLE
Add host whitelist from environment.js

### PR DESCRIPTION
**Pull request by danmcclain**
Handles regex by turning it into a string in the `package.json`, ember-fastboot-server will handle this correctly

### CODE REVIEW

**Comment by danmcclain** [comment]
@tomdale Ping

File: lib/addon-modes/fastboot.js [hunk 1]
```diff
@@ -49,13 +49,16 @@ module.exports = {
...
+
       // Create a new Broccoli tree that writes the FastBoot app's
       // `package.json`.
       var config = new FastBootConfig(tree, {
         project: this.project,
         outputPaths: this.outputPaths,
         assetMapPath: this.assetMapPath,
-        ui: this.ui
+        ui: this.ui,
+        fastBootAppConfig: this.project.config(environment).fastBoot
```
Comment (tomdale) [inline lib/addon-modes/fastboot.js]: This should be capitalized as `fastboot` rather than `fastBoot`.

Comment (danmcclain) [inline lib/addon-modes/fastboot.js]: Fixing now

**Comment by danmcclain** [comment]
@tomdale fixed capitalization, test will fail until https://github.com/tomdale/ember-cli-addon-tests/pull/6 is merged and released

**Comment by danmcclain** [comment]
@tomdale Should be good to merge

**Pull request merged**